SDALib-CPP/SDALib-Python/PythonDriver.h

```diff
@@ -6,25 +6,80 @@
 #include "SDAAction.hpp"
 #include "SDADriver.hpp"
 #include "SDAData.hpp"
-#include "boost/python.hpp"
-using namespace boost::python;
+#ifdef _DEBUG
+#undef _DEBUG
+#include <python.h>
+#define _DEBUG
+#else
+#include <Python.h>
+#endif
```

Comment:
are you only allowed to have the Release version???? Btw it is Python with a capital always

Reply:
This was the only way I was able to get it to work. It is very difficult to find debug packages for Python